erts/emulator/beam/ops.tab

```diff
@@ -77,17 +77,13 @@ return
 # To ensure that a "move Src x(0)" instruction can be combined with
 # the following call instruction, we need to make sure that there is
 # no line/1 instruction between the move and the call.
-#
-# A tail-recursive call to an external function (BIF or non-BIF) will
-# never be saved on the stack, so there is no reason to keep the line
-# instruction.
 
 move S X0=x==0 | line Loc | call_ext Ar Func => \
      line Loc | move S X0 | call_ext Ar Func
 move S X0=x==0 | line Loc | call_ext_last Ar Func D => \
-     move S X0 | call_ext_last Ar Func D
```

Comment:
Maybe add a comment here saying that external calls to bifs may be saved on stack when traced so the line info may be needed?

Reply:
On another note, shouldn't really the instr combining logic just ignore line instructions altogether so that we don't have do these types of things....

Reply:
Done, I'll look into ignoring line instructions later on.